language/move-prover/spec-lang/src/env.rs

```diff
@@ -1069,6 +1089,22 @@ impl<'env> ModuleEnv<'env> {
             })
     }
 
+    /// Returns iterator over ConstantEnvs in this module.
+    pub fn get_constants(&'env self) -> impl Iterator<Item = ConstantEnv<'env>> {
+        self.clone().into_constants()
+    }
+
+    /// Returns iterator over FunctionEnvs in this module.
```

Comment:
comment c&p

Reply:
There is ExprTranslator::translate_value.

Reply:
`ExpTranslator:: translate_value` translates `EA::Value`, not `MoveValue`